components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/Application.java

```diff
@@ -195,4 +195,13 @@ public HashMap getPermissionMap() {
     public void setPermissionMap(HashMap permissionMap) {
         this.permissionMap = permissionMap;
     }
+
+    @Override
+    public String toString() {
+        return "Application [name=" + name + ", groupId=" + groupId + ", uuid=" + uuid + ", policyId=" + policyId
```

Comment:
Usage in this method is when we enable debug logs. having toString() method would be usefull when we go through the debug logs